### PR TITLE
Make `ENDL` optional like `ENDSECTION`

### LINKED ISSUE #1534: Missing `ENDL` after `LOAD` is not an error
At the *very least* this should be a warning, and I expect it should be an error like other missing terminators (`ENDC`, `ENDR`, `ENDU`, etc).

```asm
section "rom", rom0
Label::
db 1, 2, 3
dw startof("rom"), sizeof("rom")
load "ram", wram0
wLabel::
db 4, 5, 6
dw startof("ram"), sizeof("ram")
```

### PR DESCRIPTION
**Pull request by Rangi42**
Fixes #1534

### CODE REVIEW

**Review by ISSOtm — COMMENTED** [review]
The change itself makes sense, IMO, but I'm also wondering whether we may want to introduce a `-Wunclosed-load` warning? Probably in `-Wextra`, if anywhere. (I'm not concerned about `-Wunclosed-section`, because the difference is that forgetting to close a `LOAD` can end up over-reserving RAM without producing an error, whereas directives that work inside of a `SECTION` are not affected by the presence of one.)

**Comment by Rangi42** [comment]
Done! I've added `-Wunterminated-section`.

File: contrib/bash_compl/_rgbasm.bash
Comment (ISSOtm) [inline contrib/bash_compl/_rgbasm.bash:1]: No equivalent for Zsh?

**Review by ISSOtm — APPROVED** [review]
You mean `-Wunderminated-load` ;)

Thank you for the thorough testing!

**Pull request merged**